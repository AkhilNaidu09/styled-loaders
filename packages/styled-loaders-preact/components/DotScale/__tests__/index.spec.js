const render = require('preact-render-to-string');
const DotScale = require('../');
import 'jest-styled-components';

describe('DotScale component', () => {
	it('should render', () => {
		const tree = render(<DotScale />);
		expect(tree).toMatchSnapshot();
	});

	it('should render', () => {
		const tree = render(<DotScale color="#BADA55" />);
		expect(tree).toMatchSnapshot();
	});

	it('should render', () => {
		const tree = render(<DotScale duration="0.2s" />);
		expect(tree).toMatchSnapshot();
	});

	it('should render', () => {
		const tree = render(<DotScale dotSize="50px" size="200px" />);
		expect(tree).toMatchSnapshot();
	});
});
