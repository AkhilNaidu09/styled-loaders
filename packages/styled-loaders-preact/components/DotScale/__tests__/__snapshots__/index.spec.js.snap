// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DotScale component should render 1`] = `"<[object Object]></[object Object]>"`;

exports[`DotScale component should render 2`] = `"<[object Object] color=\\"#BADA55\\"></[object Object]>"`;

exports[`DotScale component should render 3`] = `"<[object Object] duration=\\"0.2s\\"></[object Object]>"`;

exports[`DotScale component should render 4`] = `"<[object Object] dotSize=\\"50px\\" size=\\"200px\\"></[object Object]>"`;
